docs/resources/job.md

```diff
@@ -468,8 +468,8 @@ This syntax uses Jobs API 2.0 to create a job with a single task. Only a subset
 
 The job cluster is specified using either of the below argument:
 
-* `new_cluster` - (Optional) Same set of parameters as for [databricks_cluster](cluster.md) resource.
-* `existing_cluster_id` - (Optional) If existing_cluster_id, the ID of an existing [cluster](cluster.md) that will be used for all runs of this job. When running jobs on an existing cluster, you may need to manually restart the cluster if it stops responding. We strongly suggest to use `new_cluster` for greater reliability.
+* `new_cluster` - (Required) Same set of parameters as for [databricks_cluster](cluster.md) resource.
```

Comment:
exactly one of these two arguments is required, (Required) is misleading

Reply:
Is it tho? I think they are both marked required on the openapi spec, as well as on the public facing [documentation](https://docs.databricks.com/api/workspace/jobs/create#job_clusters). Does that mean the openapi spec is wrong?

Reply:
actually, this part of the documentation is related to `Single-task syntax (deprecated)`, so we should not update the doc

Reply:
Updated